src/screens/Survey/SurveyMainScreen.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Text, View, StyleSheet, Button} from 'react-native';
+import {createStackNavigator} from 'react-navigation';
+import SurveyStart from './SurveyStart';
+import Question from './redux/containers/Container_Question';
+
+var RootStack = createStackNavigator(
+  {
+    Home: {screen: SurveyStart},
+    Question: {screen: Question},
+  },
+  {
+    cardStyle : {
+      backgroundColor: 'transparent',
+      alignItems:'center',
+      justifyContent:'center'
+    },
+    headerMode: 'none',
+    mode: 'card',
+    swipeEnabled: true,
+    navigationOptions: () => ({
+      gesturesEnabled : true,
+      gestureResponseDistance: {
+        horizontal: 200
+      }
+    }),
+    transitionConfig: () => ({
```

Comment:
What effect does this custom transition achieve?

Reply:
The default animation goes from the bottom to the top. This config makes it go from right to left